packages/ocaml-freestanding/ocaml-freestanding.0.2.3/opam

```diff
@@ -20,6 +20,6 @@ conflicts: [
 ]
 available: [
   ocaml-version >= "4.02.3" & ocaml-version < "4.07.0" &
-  ((os = "linux" & (arch = "x86_64" | arch = "aarch64")) |
+  ((os = "linux" & (arch = "x86_64" | arch = "arm64")) |
```

Comment:
@kit-ty-kate Based on the discussion in https://github.com/mirage/ocaml-freestanding/pull/36, this doesn't seem right. `ocaml-freestanding` _does_ work on Linux/aarch64, won't this break users trying to install it with OPAM 1 where `arch` will be `aarch64`?

Reply:
Yes yes, I haven't made the change yet. I'll do today.

Reply:
By change I mean "don't change anything"